From d1c247b6002120885cbd0d234931b93a6d04851c Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sun, 11 Feb 2018 20:29:43 +0800
Subject: [PATCH] sound/soc/codecs/rt5651: fix compile warning

sound/soc/codecs/rt5651.c:351:14: warning: duplicate const
declaration specifier [-Wduplicate-decl-specifier]

Change-Id: I25ac1012398c7742c7b66d6a57f4956b96ba0aa4
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 sound/soc/codecs/rt5651.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/sound/soc/codecs/rt5651.c b/sound/soc/codecs/rt5651.c
index 7de785087958..d50ee82bc904 100644
--- a/sound/soc/codecs/rt5651.c
+++ b/sound/soc/codecs/rt5651.c
@@ -348,8 +348,7 @@ static SOC_ENUM_SINGLE_DECL(rt5651_if2_adc_enum, RT5651_DIG_INF_DATA,
 
 static const char * const rt5651_asrc_mode[] = {"Disable", "Enable"};
 
-static const SOC_ENUM_SINGLE_DECL(rt5651_asrc_enum, 0, 0,
-		rt5651_asrc_mode);
+static SOC_ENUM_SINGLE_DECL(rt5651_asrc_enum, 0, 0, rt5651_asrc_mode);
 
 static const struct snd_kcontrol_new rt5651_snd_controls[] = {
 	/* Headphone Output Volume */
-- 
2.35.3

